### PR TITLE
bsp: linux-lmp-fslc-imx: update to include the lf-6.1.55-2.2.0 tag

### PR DESCRIPTION
**Pull request by MrCry0**
Update branch and stable-base version to include the lf-6.1.55-2.2.0 tag.

### CODE REVIEW

File: meta-lmp-base/recipes-bsp/u-boot/u-boot-fio_imx-2023.04.bb [hunk 1]
```diff
@@ -1,7 +1,8 @@
 require u-boot-fio-common.inc
 
-SRCREV = "514446b012926b395b7f08cd3b28c82a2f303a9f"
-SRCBRANCH = "2023.04+lf-6.1.36-2.1.0-fio"
+UBOOT_REPO = "git://github.com/MrCry0/u-boot.git"
```
Comment (quaresmajose) [inline meta-lmp-base/recipes-bsp/u-boot/u-boot-fio_imx-2023.04.bb:3]: seems to have been forgotten here

Comment (MrCry0) [reply meta-lmp-base/recipes-bsp/u-boot/u-boot-fio_imx-2023.04.bb:3]: Seems I need to merge kernel before :-D

Comment (MrCry0) [reply meta-lmp-base/recipes-bsp/u-boot/u-boot-fio_imx-2023.04.bb]: > seems to have been forgotten here

Thanks, @quaresmajose Fixed!

**Review by quaresmajose — APPROVED** [review]

**Comment by ricardosalveti** [comment]
Looks good, just please hold until we release v93.

**Review by quaresmajose — APPROVED** [review]

**Pull request merged**